turbo.json

```diff
@@ -8,7 +8,7 @@
 		"enabled": false,
 		"signature": true
 	},
-	"ui": "stream",
+	"ui": "tui",
```

Comment:
tui seems pretty nice now and is required for interactive tasks like `preview` and `dev`